examples/api/test/material/snack_bar/snack_bar.1_test.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/snack_bar/snack_bar.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Tapping on button shows snackbar', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.SnackBarExampleApp(),
+    );
+
+    expect(find.byType(SnackBar), findsNothing);
+    expect(find.widgetWithText(AppBar, 'SnackBar Sample'), findsOneWidget);
+    expect(find.widgetWithText(ElevatedButton, 'Show Snackbar'), findsOneWidget);
+    await tester.tap(find.widgetWithText(ElevatedButton, 'Show Snackbar'));
```

Comment:
```suggestion

    await tester.tap(find.widgetWithText(ElevatedButton, 'Show Snackbar'));
```

Tap action already finds the buttons